### ISSUE TITLE
one Echo does not launch in the HMD

### ISSUE THREAD

**Issue by digicraft63** [LibreVR/Revive#1557]
**Describe the bug**
Lone Echo does not launch in the HMD. Only on the monitor. 

**To Reproduce**
1. Start piTool
2. Start Pimax 8k Headset
3. Start SteamVr
4. Start Revive Dashboard
5. Start Lone Echo from Dashboard
6. Lone Echo loading shows in the HMD but the game is loading only on the monitor instead of HMD

**Expected behavior**
I expected Lone Echo to play in the HMD and not only on the monitor.

**Environment (please complete the following information):**
 - App: Lone Echo
 - OS: Win 10
 - Headset: Pimax 8k

**Versions (please complete the following information):**
 - Revive: 2.0.0.13
 - SteamVR: 1.11.13

**Create a system report**
[SteamVR-2020-05-28-AM_01_33_26.txt](https://github.com/LibreVR/Revive/files/4692416/SteamVR-2020-05-28-AM_01_33_26.txt)

**Additional context**
Tracking of Vive controllers and the HMD is working but only displayed on the monitor.
[SteamVR-2020-05-28-AM_01_33_26.txt](https://github.com/LibreVR/Revive/files/4692427/SteamVR-2020-05-28-AM_01_33_26.txt)

**Comment by crystallon** [comment]
Same problem on Pimax Artisan with Lone Echo and Stormland.
In same time Dreamdesk, First contact, Oculus home working fine.

**Comment by patricklupo** [comment]
Same here on Pimax 8KX
Lone Echo starts, runs only on monitor screen with index controllers working but just says Loading in HMD
Windows 10 1904
Revive 2.1.1.0
SteamVR 1.15.15
PiTool 1.0.1.266
Latest Oculus Redistributable

**Comment by digicraft63** [comment]
Well, it works for me if you start Lone Echo from within the PiTool.
Doesn´t work if you try to start it from SteamVR.

**Comment by Nostrildumbass** [comment]
Hi, yes, however then you have no Chaperone boundaries, which I tend to
need for a game like Echo Arena/Combat. I've smacked my walls quite a few
times in those XD

**Comment by ziarno** [comment]
Having the same issue on Pimax 8kx. Not just for Lone Echo, happened for more games too.
@digicraft63 the game crashes for me when I try to run it rom piTool, did you do something special to get it working?

**Comment by Nostrildumbass** [comment]
Have you tried lowering the FOV and enabling parallel projections? That
lets me run Echo from PiTool at least, but it doesn't let me play Wrench :(

**Comment by ziarno** [comment]
Yes, it was the Large FOV that was causing the crash, thx @Nostrildumbass!

**Comment by digicraft63** [comment]
Good that you found a solution.

However Lone Echo has a tendency to crash here and there during game play. And to that I have found no solution so far.

**Comment by Nostrildumbass** [comment]
Bump
